### DISCUSSION TITLE
Zig 开发常用类库

### DISCUSSION THREAD

**Discussion by jiacai2050** [zigcc/forum#28] [观点交流]
# 命令行工具
- https://github.com/Sobeston/table-helper 以表格形式打印
- https://github.com/Hejsil/zig-clap 命令行参数解析
- https://git.sr.ht/~leon_plickat/zig-spoon a zig library offering a simple, low-level and allocation free abstraction for creating TUI programs. 
# 多线程
- https://github.com/Jack-Ji/zig-async
- https://github.com/kprotty/zap An asynchronous runtime with a focus on performance and resource efficiency.

# Web 开发
- https://github.com/haze/zelda A simple HTTP client library for Zig

# 编码
- https://github.com/kubkon/protozig The protozig(uana), or protocol buffers implementation in Zig

# 模板引擎
- https://github.com/haze/etch zig template framework

# C 库 Wrapper
- https://github.com/mitchellh/zig-libxml2
- https://github.com/vrischmann/zig-sqlite
- https://github.com/mattnite/zig-zlib

# 其他
欢迎大家补充

**Comment by hhstore** [comment]
# zig 生态: 

- https://github.com/hhstore/blog/issues/374

**Reply by jiacai2050** [reply]
赞，记得之前在哪里看到过这个，搜了下没找到就简单列了下

**Reply by hhstore** [reply]
群里贴过

**Comment by jiacai2050** [comment]
- https://zig.pm/
社区内已经有个解决方案了，后面有新包可以尝试增加到 https://github.com/ziglibs/repository

**Comment by jiacai2050** [comment]
Move to https://github.com/zigcc/awesome-zig
